docs/en_US/network-dumps.md

```diff
@@ -0,0 +1,69 @@
+# Network Dumps
+One of the best ways to support the project is to help in gathering network dumps of games. These network dumps can be used by developers to help understand how the servers operate in a live environment. This kind of data greatly simplifies development of the servers, as it removes a large amount of guesswork and reverse engineering.
+
+In order to make this easier, we have developed a suite of internal tools to capture and read network traffic, as well as make use of several "off the shelf" tools. This document will go into detail about how to capture these packets yourself, and how you can submit them to us for use in research and development. This document will be fairly long, please read it completely before submitting any dumps.
+
+## Table of Contents
+1. [Security Warning](#security-warning)
+2. [Submissions](#submissions)
+3. [Game Packets](#game-packets)
+	- [Wii U (HokakuCafe)](#wii-u-hokakucafe)
+	- [3DS (HokakuCTR)](#3ds-hokakuctr)
+	- [All (WireShark)](#all-wireshark)
+4. [HTTP Packets](#http-packets)
+	- [Fiddler](#fiddler-windows-free)
+	- [Charles](#charles-cross-platform-50)
+	- [Mitmproxy](#mitmproxy-cross-platform-free)
+
+# Security Warning
+## Network dumps will oftentimes have sensitive information, including emails, passwords, usernames, IP addresses, etc. Because of this, ensure you only provide network dumps to trusted individuals. When submitting network dumps to Pretendo Network developers your dumps are uploaded and stored in a private channel. This channel can only be accessed by a small number of people, not even all developers have access.
```

Comment:
We could probably use a red tip, similar to the Cemu page

![imagen](https://github.com/PretendoNetwork/website/assets/112760654/80dc2675-bbac-4746-be60-6a4d1f59f3ba)

Reply:
I forgot we had those, good catch